ci/jobs/fast_test.py

```diff
@@ -257,16 +254,19 @@ def main():
                 )
             )
         if not results[-1].is_ok():
-            attach_files.append(f"{temp_dir}/build/programs/clickhouse")
-            attach_files += [
-                f"{temp_dir}/var/log/clickhouse-server/clickhouse-server.err.log",
-                f"{temp_dir}/var/log/clickhouse-server/clickhouse-server.log",
-            ]
+            attach_debug = True
+
+    if attach_debug:
```

Comment:
Attaching files only in case of failures is an interesting idea, for ourselves we have ci-logs, for others, they will likely need them only in case of failures (though I sometimes compare the logs, and always considered it as "feature" of ClickHouse CI)

But it is OK to me (at least for fasttest)